lib/plugins/digging.js

```diff
@@ -166,6 +166,7 @@ function inject (bot) {
     await diggingTask.promise
   }
 
+  bot.on('death', () => { bot.removeAllListeners('diggingAborted'); bot.removeAllListeners('diggingCompleted') })
```

Comment:
can you do this in multiple lines ?

Reply:
more things need to be done to cancel digging, see https://github.com/PrismarineJS/mineflayer/pull/1850/files#diff-b89259c2ab8d812e6fbb1bc79808cf590b9417dc35b7848f45bb96f96d87f927R131

Reply:
The link just leads to the Files Changed section of this pull request.